include/contin_array.h

```diff
@@ -9,20 +9,30 @@
 
 #include <zephyr/kernel.h>
 
-/** @brief. Creates a continuous array from a finite array
- * @param pcm_cont		Pointer to the destination array
+/**
```

Comment:
Suggestion: This function takes an array which the user wants to loop over (could e.g. be a period of a Pulse-code modulated (PCM) sine wave). This is supplied to pcm_finite and pcm_finite_size. The user may then call this function which populates pcm_cont with pcm_cont_size bytes. This lets users create N repetitions of pcm_finite into pcm_cont.
The function keeps track of the current position in pcm_finite so the function can be called multiple times and maintain the correct position in pcm_finite. Wrapping is handled automatically.